tox.ini

```
@@ -10,13 +10,17 @@ commands = test_plugin --standalone plugin.py
 commands = test_plugin --standalone plugin.py --regenerate
 
 [testenv:package]
-commands = build_plugin plugin.py . {posargs}
+deps = hansken-extraction-plugin==0.8.3
+commands = build_plugin . {posargs}
```

Would it make sense to parametrize the dot as well, or is more of it sensitive to the current working directory?

```suggestion
commands = build_plugin {tox_root} {posargs}
```

---

would make sense. I tried your suggestion, however it seems that `tox_root` is not available yet on Ubuntu 22.04 default tox version (3.21.4). Since that is some of still have to deal with, I'll leave the dot for now. :(